backend/clubs/migrations/0114_alter_advisor_public.py

```diff
@@ -0,0 +1,38 @@
+# Generated by Django 5.0.4 on 2024-09-29 17:00
+
+from django.db import migrations, models
+
+
+def migrate_public_to_enum(apps, schema_editor):
+    Advisor = apps.get_model("clubs", "Advisor")
+    # Update 'public' field, assume public=True means only students by default
+    Advisor.objects.filter(public_old=False).update(public=1)
+    Advisor.objects.filter(public_old=True).update(public=2)
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("clubs", "0113_badge_message"),
+    ]
+
+    operations = [
+        migrations.RenameField(
+            model_name="advisor",
+            old_name="public",
+            new_name="public_old",
+        ),
+        migrations.AddField(
+            model_name="advisor",
+            name="public",
+            field=models.IntegerField(
+                choices=[(1, "Admin Only"), (2, "Signed-in Students"), (3, "Public")],
+                default=2,
+            ),
+        ),
+        migrations.RunPython(migrate_public_to_enum),
```

Comment:
Can you please define the `reverse_code` argument in case we need to undo this migration? See [here](https://docs.djangoproject.com/en/5.1/ref/migration-operations/#runpython)